### ISSUE TITLE
✔️ 🐞 Bug Report: Global fonts always calls fonts.googleapis.com 

### ISSUE THREAD

**Issue by colmtroy** [elementor/elementor#16023]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

The default `elementor_scheme_typography` controls cause an unnecessary fonts.googleapis.com http request that has speed and performance implications and cannot be set to be switched off by default no matter what the user does.

Goal: Complete removal of all http requests to Google font servers.

## Steps to reproduce

1.) Clean install of Elementor (3.4.0), WordPress (5.8). Default theme: Twenty Twenty-One
2.) Disable Default Colors and Fonts in Elementor -> Settings
3.) Create new Page in Pages -> Add new
4.) Add a heading and a text editor. Observe that typography is set to "Default"
![](https://i.imgur.com/cwwiHIc.png)
5.) Publish page and open it.
6.) Open network tab in Google Chrome dev tools and search for "fonts.googleapis.com". Observe that a completely unnecessary http request is sent to fonts.googleapis.com
![](https://i.imgur.com/dF7A8vd.png) 

The root cause of this issue is the Site Settings -> Global Fonts trigger Roboto, Robot Slab et. al. to be called even though we disabled Default fonts in Step 2 above.
 
![](https://i.imgur.com/SwBVFBn.png)

The `elementor_scheme_typography` DB option looks like this despite Elementor fonts being disabled globally.
![](https://i.imgur.com/Wg5RuHh.png )

This additional http request is very expensive for those of us conducting webperf optimization. The only way to avoid it is to manually update Site Settings -> Global Fonts and set it to use "Default" too manually, but this is a problem for lots of reasons:
1.) It's not at all clear that the user needs to do this to avoid the fonts.googleapis.com http request
2.) As far as the user is concerned, they already completed this step in Step 2 above.
3.) The only way to make this change is by first creating a page - so it's not at all obvious to the user that they need to update a global typography scheme option at a page level editor.

I would strongly recommend 2 things:
a.) Update the `elementor_scheme_typography` DB option to remove the Elementor Typography scheme default fonts when a user completes Step 2 above.
b.) Allow theme authors/plugin devs to set/filter `elementor_scheme_typography` so as to set this upon installation of a theme.

## Isolating the problem

- [ x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ x] This bug happens with a default WordPress theme active.
- [ x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```

== Server Environment ==
	Operating System: Darwin
	Software: nginx/1.16.0
	MySQL version: MySQL Community Server - GPL v8.0.16
	PHP Version: 7.3.5
	PHP Max Input Vars: 4000
	PHP Max Post Size: 1000M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.8
	Site URL: http://localhost:10003
	Home URL: http://localhost:10003
	WP Multisite: No
	Max Upload Size: 300 MB
	Memory limit: 256M
	Permalink Structure: Plain
	Language: en-US
	Timezone: 0
	Admin Email: you@example.com
	Debug Mode: Active

== Theme ==
	Name: Twenty Twenty-One
	Version: 1.4
	Author: the WordPress team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 3.4.0
		Author: Elementor.com


== Elementor - Compatibility Tag ==
	


== Elements Usage ==
	
	wp-page : 2
		heading : 1
		text-editor : 1


== Log ==
	


```
</details>

**Comment by black-eye** [comment]
+1
All this Roboto default stuff is a huge pain! The other side effect is that when you have the fonts defined via a theme (and not via Elementor Global Fonts), reseting any widget's font setting leads to selecting Roboto (even though it's never used on the site at all). The oly way, how to get rid of it is to redefine the theme's font setting in Elementor (by setting it either to the same font or to "default"). 

This step is really unnecessary, the default option should be "default" and not Roboto. At least when default fonts are disabled.

**Comment by oliveandlake** [comment]
I notice nobody from Elementor has eyes on this thread! @shilo-ey?

I can recreate this issue too. 

I have only Hello theme, E and E Pro installed. No other plugins.

Go to site settings, change all to Open Sans. 

Regenerate CSS. 

Still, Roboto is there. No header, no footer. Just one H1 Element which is Open Sans. There is not roboto anywhere, but it's in the GT Metrix Waterfall. I've disabled all default fonts. I can be 100% sure that there is no Roboto anywhere! :)

![image](https://user-images.githubusercontent.com/56640831/131684686-516275b3-e17d-4de2-9de6-11912770914b.png)

**Comment by murshed** [comment]
Any solution?

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor v3.10! 🥳 

✔️ Please feel free to check it out and update your plugin to the new version!

Cheers 🥂
